tests/Geocoder/Tests/Provider/GoogleMapsTest.php

```diff
@@ -196,60 +196,61 @@ public function testGeocodeBoundsWithRealAddressForNonRooftopLocation()
     public function testGeocodeWithRealAddressReturnsMultipleResults()
     {
         $provider = new GoogleMaps($this->getAdapter());
-        $results  = $provider->geocode('Paris');
+        $results  = $provider->geocode('i');
```

Comment:
If I let Paris, the test does not pass anymore because of https://maps.googleapis.com/maps/api/geocode/json?address=Paris%22 returning just one element